types/swift/WebScreen.swift

```diff
@@ -0,0 +1,122 @@
+//
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+// GENERATED FILE, DO NOT EDIT. FOR MORE INFORMATION VISIT
+// https://github.com/matrix-org/matrix-analytics-events/
+
+/// Triggered when the user changed screen on Element Web/Desktop
+extension AnalyticsEvent {
+    public struct WebScreen: AnalyticsEventProtocol {
+        public let eventName = "$pageview"
+
+        public let $current_url: $current_url
```

Comment:
I am pretty sure this is not valid Swift as well.

Reply:
You are right, I missed this when I reviewed on GH
> the '$' prefix is reserved for implicitly-synthesized declarations